plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -95,7 +95,7 @@ const config = {
 		{
 			name: 'ui',
 			use: { ...devices[ 'Desktop Chrome' ] },
-			testIgnore: '**/api-tests/**',
+			testMatch: [ '**basic.spec.js', '**/product-*spec.js' ],
```

Comment:
Updated this to have faster feedback in CI.
To revert before merging. 
```suggestion
			testIgnore: '**/api-tests/**',
```